### PR TITLE
[Added] Discord Webhook Notification (#81)

### PR DESCRIPTION
**Pull request by ifly-leonard**
Similar to the slack notification URL, added a discord notification URL. 

Screenshot 2023-01-28 at 9 07 23 PM

### CODE REVIEW

**Comment by jeffrey-zang** [comment]
yooooo this looks great

**Comment by ifly-leonard** [comment]
> yooooo this looks great

tenks lel. status on revu?

Formal response: I appreciate your response. Is there any update on the PR review? I know a lot of folks (including me) who are waiting for this feature to come online.

File: resources/js/components/open/forms/components/form-components/components/FormNotificationsDiscord.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      @click.prevent="showModal=true"
+    >
+      <div class="flex-grow flex items-center">
+        <!-- <svg xmlns="http://www.w3.org/2000/svg" class="w-5 h-5 inline" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M14.5 10c-.83 0-1.5-.67-1.5-1.5v-5c0-.83.67-1.5 1.5-1.5s1.5.67 1.5 1.5v5c0 .83-.67 1.5-1.5 1.5z" /><path d="M20.5 10H19V8.5c0-.83.67-1.5 1.5-1.5s1.5.67 1.5 1.5-.67 1.5-1.5 1.5z" /><path d="M9.5 14c.83 0 1.5.67 1.5 1.5v5c0 .83-.67 1.5-1.5 1.5S8 21.33 8 20.5v-5c0-.83.67-1.5 1.5-1.5z" /><path d="M3.5 14H5v1.5c0 .83-.67 1.5-1.5 1.5S2 16.33 2 15.5 2.67 14 3.5 14z" /><path d="M14 14.5c0-.83.67-1.5 1.5-1.5h5c.83 0 1.5.67 1.5 1.5s-.67 1.5-1.5 1.5h-5c-.83 0-1.5-.67-1.5-1.5z" /><path d="M15.5 19H14v1.5c0 .83.67 1.5 1.5 1.5s1.5-.67 1.5-1.5-.67-1.5-1.5-1.5z" /><path d="M10 9.5C10 8.67 9.33 8 8.5 8h-5C2.67 8 2 8.67 2 9.5S2.67 11 3.5 11h5c.83 0 1.5-.67 1.5-1.5z" /><path d="M8.5 5H10V3.5C10 2.67 9.33 2 8.5 2S7 2.67 7 3.5 7.67 5 8.5 5z" /></svg> -->
```
Comment (JhumanJ) [inline resources/js/components/open/forms/components/form-components/components/FormNotificationsDiscord.vue:8]: Can you please clear this out?

**Review by JhumanJ — CHANGES_REQUESTED** [review]
It's perfect, thank you! I just added one comment to remove a commented svg.
Thanks!

**Comment by ifly-leonard** [comment]
Copy. I clicked resolve. That's about it? First time contributing to OSS.

File: resources/js/components/open/forms/components/form-components/components/FormNotificationsDiscord.vue (hunk 1 above)
Comment (JhumanJ) [reply resources/js/components/open/forms/components/form-components/components/FormNotificationsDiscord.vue:8]: Hey @ifly-leonard I should be the one to click "resolve" 😃 It's kind of a checklist for the reviewer. More about [this here](https://www.danclarke.com/resolving-pr-comments). Please ping me when you had the chance to remove the commented out SVG, and I'll merge and deploy this! 🙌

**Review by JhumanJ — APPROVED** [review]

**Pull request merged**